src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -295,114 +591,220 @@ td.main, td.menu {
     border: 0;
     vertical-align: top;
 }
-.search {
-    float: right;
-    padding-top: 20px;
+
+#timeline-control {
+    background: #FFF;
+    margin: 1em 0;
+    padding: 10px;
 }
-.search label {
-    line-height: 28px;
-    vertical-align: middle;
+#timeline-control label {
+    font-size: 14px;
+    font-weight: bold;
+    margin-right: 1em;
 }
-.search input {
-    width: 195px;
+#timeline-control input {
+    font-size: 16px;
+    padding: 4px;
+    text-align: right;
+    width: 40px;
+}
+#timeline-control .help {
+    margin-left: 1em;
+}
+
+.sf-profiler-timeline .legends {
     font-size: 12px;
-    border: 1px solid #dadada;
-    background: #FFF url(data:image/gif;base64,R0lGODlhAQAFAKIAAPX19e/v7/39/fr6+urq6gAAAAAAAAAAACH5BAAAAAAALAAAAAABAAUAAAMESAEjCQA7) repeat-x left top;
-    padding: 5px 6px;
-    color: #565656;
+    line-height: 1.5em;
 }
-.search input[type="search"] {
-    -webkit-appearance: textfield;
+.sf-profiler-timeline .legends span {
+    border-left: solid 14px;
+    padding: 0 10px 0 5px;
 }
-#navigation div:first-child {
-    margin: 0 0 20px;
-    border-top: 0;
+.sf-profiler-timeline canvas {
+    border: 1px solid #CCC;
+    background: #FFF;
+    margin: .5em 0;
+}
+
+
+{% verbatim %}
```

Comment:
Without the verbatim tag, the selector `#tabs input[type=radio]:checked + label` produces an error.

But don't worry about this. The problem is that I dreamed about creating a "pure CSS" design. The tabs and the search form toggle are implemented just with CSS. However, this requires some ugly CSS tricks so the dream turned into a nightmare. I'm going to update this to use JavaScript.